services/web/server/tests/integration/test_exporter.py

```diff
@@ -191,6 +191,7 @@ async def login_user(client):
 
 
 def get_exported_projects() -> List[Path]:
+    # TODO: explain how to re-generate these files?
```

Comment:
If you unzip them and have a look inside, they are 2 projects with the same structure, containing different versions of the sleeper. The only important thing is the labels of the nodes. They have to follow this pattern:`name_number` like: (sleeper_0, sleeper_1, etc...)
Also these are meant not to be regenerated as they are the "decided" format produced by FormatterV1 and FormatterV2

Reply:
@GitHK I understand that, but do you have a script to generate them?

Reply:
No, and we **should not** have one. They needn't be regenerated. If a new format is released a version3 will be added.